worldedit-sponge/build.gradle

```diff
@@ -35,8 +35,8 @@ sponge {
 }
 
 minecraft {
-    version = "1.8.9"
-    mappings = "stable_22"
+    version = "1.10.2"
+    mappings = "snapshot_20160629"
```

Comment:
I copied it from the forge version. thought it'd make sense to keep parity between the two.